src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -202,6 +202,20 @@ public static partial class RoslynInsertionTool
                     }
                 }
 
+                // ********************* Trigger a release *****************************
+                Log.Info($"Triggering a release for the build {newestBuild.BuildNumber}");
+
+                var release = await CreateReleaseAsync(newestBuild, cancellationToken);
+
+                // The timeout for the below wait is primarily dependent on:
```

Comment:
I honestly wouldn't wait more than 5 minutes or so. It's more important for us to get the PR going so we can start val builds than anything else, I'd think.

Reply:
I can reduce this timeout to 5/10 minutes but it would fail in cases when the build queue is swamped with other requests already. Two ways of getting out that though:
1. Have a dedicated always available pool.
2. Figure out a way of kicking of the release definition way earlier when the build itself is complete. We could achieve this selectively via tags. Since the build triggered by vslsnap would mostly be triggered once a day(during the insertion changes) we wouldn't be swamping the symbol archival. That could also enable others to selectively auto-trigger release when required.

Reply:
Reduced the timeout. Will monitor this over the next few days to see if we run into any spikes like I saw yesterday night.

Reply:
And the symbol check seems to be auto-triggering today(which does make sense), so it looks like there isn't a way to delay waiting for the release to complete after creating a PR.